lib/File/Find/Rule.pm

```diff
@@ -485,6 +485,18 @@ sub relative () {
     $self;
 }
 
+=item C<normalize>
+
+Normalize paths found using C<File::Spec->canonpath>
```

Comment:
I guess it can be called "canonise" or "canonpath" since all this does is process the path through `File::Spec->canonpath()`. Let me know what your preference is.

Also, I'm not sure if it is apt to document here what another module's method does. AFAIK, `File::Spec` has been around forever, and `canonpath` is pretty widely used and known.